articles/fingerprint-authentication-in-android-applications/index.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/fingerprint-authentication-in-android-applications/
+title: Impleemening Fingerprint Authentication in Android applications
+description: In this article, we will learn how to implement fingerprint authentication in an Android application.
+author: briana-nzivu
+date: 2021-05-07T00:00:00-08:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/fingerprint-authentication-in-android-applications/hero.jpg
+    alt: Fingerprint authentication
+---
+
+Biometrics are calculations and body measurements that are related to human characteristics. Biometrics is categorized into: Physical and behavioral. Physical biometrics is our humanly features such as our fingerprints, iris, and many others. Behavioral is our actions, for example, smiling, hand signs, and others.
+<!--more-->
+In turn, biometrics has brought up vast, easy, and secure ways to identify a specific user using their physical traits.
+
+### Introduction
+Let us admit it, typing over eight characters in the name of a password is quite a task, that is if a user even remembers the password. These days most developers and companies have made it easier for us by using biometrics as a form of authentication. Biometrics, time and again, has proved to be more efficient and less prone to cyber-attacks. Most fields such as Banking, Information Technology, Education, and many others have implemented biometrics into their systems for various purposes. This article will cover one of the biometric ways of authentication, which is using our fingerprint.
+
+#### Advantages of Biometrics
+- Biometrics is specific because it is associated with a single person (unlike a password that anyone can authorize).
+- It is very convenient. There is no need to carry or remember anything.
+- It is very secure. Biometrics is highly fraud-resistant.
+
+#### Disadvantages
+- They are error-prone, sometimes even causing the system to shut down; hence access is limited.
+- Biometric devices and in-built biometric features are costly compared to other traditional devices.
+
+
+### Prerequisites
+- Prior experience in building basic Android applications.
+- Basic knowledge of XML and Kotlin programming languages.
```

Comment:
XML does not qualify to be a programming language as it does not perform any computation or algorithms.